numpy/f2py/setup.py

```diff
@@ -26,6 +26,9 @@
 def configuration(parent_package='', top_path=None):
     config = Configuration('f2py', parent_package, top_path)
     config.add_subpackage('tests')
+    config.add_subpackage('backends')
+    config.add_data_dir('backends/src')
```

Comment:
I don't think this does anything, the next line covers it already. You should need one or the other I believe.